example/package.json

```diff
@@ -3,6 +3,8 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
+    "dev:standalone": "cross-env MODE=standalone vite",
```

Comment:
I think lets replace the "start:standalone"/"start:remote" commands with these ones, not sure why we would want to keep the old versions.